knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-resources.yaml

```diff
@@ -64,7 +64,7 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\", pod=~\"$sprefix-$source-.+\", container != \"POD\", container != \"\"}[1m])) by (container)",
+              "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\", pod=~\"$sprefix-$source-.+|pingsource-mt-adapter-.+\", container != \"POD\", container != \"\"}[1m])) by (container)",
```

Comment:
@skonto can you explain the change here? regex here is hard to read

Reply:
Yes I need to monitor the pingsource-mt-adapter in knative-eventing namespace.
This cannot be covered by the pattern for the other resources which is:
```
$sprefix-$source-
apiserversource-testevents-f61cc82f-11e2-41d6-9ce9-9242a3cs29cm
sprefix = apiserversource
source = testevents
```

Reply:
thanks